src/pkg/backup/details/details.go

```diff
@@ -89,21 +92,48 @@ func (dm DetailsModel) Paths() []string {
 
 // Items returns a slice of *ItemInfo that does not contain any FolderInfo
 // entries. Required because not all folders in the details are valid resource
-// paths.
+// paths, and we want to slice out metadata.
 func (dm DetailsModel) Items() []*DetailsEntry {
 	res := make([]*DetailsEntry, 0, len(dm.Entries))
 
 	for i := 0; i < len(dm.Entries); i++ {
-		if dm.Entries[i].Folder != nil {
+		ent := dm.Entries[i]
+		if ent.Folder != nil || ent.isMetaFile() {
 			continue
 		}
 
-		res = append(res, &dm.Entries[i])
+		res = append(res, &ent)
 	}
 
 	return res
 }
 
+// SliceMetaFiles returns a copy of the Details with all of the
+// .meta files sliced out of the entries.
+func (dm DetailsModel) SliceMetaFiles() DetailsModel {
+	d2 := DetailsModel{
+		Entries: []DetailsEntry{},
+	}
+
+	for _, ent := range dm.Entries {
+		if !ent.isMetaFile() {
+			d2.Entries = append(d2.Entries, ent)
+		}
+	}
+
+	return d2
+}
+
+func (de DetailsEntry) isMetaFile() bool {
+	// TODO: this is hazardous.  legitimate files could be suffixed .meta.
```

Comment:
We need to do this based on the backup version. For backup version 0, there is never any meta files. All files, even the ones that end with `.meta` are data files and for all other versions, if the suffix is `.meta` or `.dirmeta`, they are metadata files.

Reply:
My concern is that if we have files from older backup with a suffix of `.meta` in the original file name, it can end up getting treated as a metadata file even though it is not.

Reply:
Agreed, thus the todo.  I can add a known issue to the changelog, if that helps.  Also, not saying v2 needs to be a long follow.  You should probably tail this PR with the change immediately.  But I think this is safe enough for the immediate case, and it's too late for me to push that next part.